DotNetWebApi/Controllers/PicnicController.cs

```diff
@@ -0,0 +1,138 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using DotNetWebApi.Models;
+
+namespace DotNetWebApi.Controllers;
+
+[Route("api/")]
+[ApiController]
+public class PicnicController : ControllerBase
+{
+    private readonly TeddyBearsContext _context;
+
+    public PicnicController(TeddyBearsContext context)
+    {
+        _context = context;
+    }
+[HttpGet("Picnics")]
```

Comment:
Nit: The indentation here should match the method definition.  Also, there is typically a line of whitespace between methods.